.github/workflows/deploy.yml

```diff
@@ -24,3 +24,88 @@ jobs:
           cloudflare_api_token: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           commit_sha: ${{ github.event.workflow_run.head_sha }}
           workflow_run_id: ${{ github.event.workflow_run.id }}
+
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.10.0
+
+      - name: Generate Claimable Permit
+        id: permit_generation
+        # The sed command is used to remove color characters that are present in the output of the script
+        run: |
+          yarn
+          output=$(yarn "start:sign")
+          url=$(echo $output | grep -o "https://[^ ]*" | sed -n '2p' | sed 's/\x1B\[[0-9;]*[JKmsu]//g')
+          echo $output
+          echo "Permit available at the address:"
+          echo $url
+          echo "CLAIMABLE_URL=$url" >> $GITHUB_ENV
+        env:
+          BENEFICIARY_ADDRESS: "0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd"
+          PAYMENT_TOKEN_ADDRESS: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
+          AMOUNT_IN_ETH: 0
```

Comment:
I just realized we could make a faucet service if we sent like 0.0005 WXDAI. This should allow for about 3 transactions.

This could be an interesting plugin for new contributors!